src/imitation/rewards/reward_nets.py

```diff
@@ -239,6 +242,31 @@ def base(self) -> RewardNet:
         return self._base
 
 
+class RewardNetWithVariance(RewardNet):
+    """A reward net that keeps track of its epistemic uncertainty through variance."""
+
+    @abc.abstractmethod
+    def reward_moments(
```

Comment:
Other functions that take numpy inputs are prefixed with `predict_`, do we want to stick to that convention?

Reply:
I think it's a reasonable convention. I've switched it over to `predict_reward_moments`.